src/app/services/project_service.js

```diff
@@ -280,6 +304,17 @@ angular
             })
         }
 
+		try {
+			//As search terms become longer, be less tolerant of tiny fuzzy matches
+			var shortestWord = q.split(' ').sort(function(a, b){ return a.length - b.length})[0]
+			service.fuse.options.minMatchCharLength = Math.max(1, shortestWord.length - 2);
```

Comment:
I _think_ this works. I found that you can't modify the `options.keys` list after creation, presumably because it's calculating the index once, but it looks like this can toggle back and forth without trouble.

Reply:
huh... did you find that search results were _too_ fuzzy before adding this code? I wonder it ratcheting down the `threshold` will help with this without needing to hack into the search options for each keystroke

Reply:
Yeah, I found it distracting to have single characters highlighted in amongst the main word. Compare this:
![Screen Shot 2021-02-10 at 11 26 48 PM](https://user-images.githubusercontent.com/7335046/107497575-a2f52100-6bf7-11eb-968b-451ab670b103.png)

with 
![Screen Shot 2021-02-10 at 11 27 39 PM](https://user-images.githubusercontent.com/7335046/107497589-a7213e80-6bf7-11eb-8541-b474d491e463.png)

This is after changing the threshold to 0.4. 

With that said, it looks like it might be costing ~40ms (back of the envelope) so might have to give up on it.